CalculatorExercises/1-CalculatorBasic/Calculator.java

```diff
@@ -0,0 +1,100 @@
+import java.util.*;
+import java.lang.*;
+
+public class Calculator {
+
+    enum MathOperator {
+        SUM, SUBSTRACTION, DIVIDE, MULTIPLCATION;
+
+    }
+
+    public static void removeWhiteSpace (StringBuilder string) {
```

Comment:
removeWhiteSpace is not Calculator specific function, create class named Utils / StringHelper / StringMethods(or something like that) that will contain methods that makes working with Strings easier.